applications/mlflow/tests/mlflow_test.py

```diff
@@ -0,0 +1,267 @@
+#!/usr/bin/env python3
```

Comment:
If you're going to use python for testing, wouldn't using pytest clean things up some? It would handle printing test results, provide a modular system to define tests, and you can just lean into the other python tooling for dependencies and environments.

Reply:
As above, I'm fine with testing being a follow up task with a note somewhere that it's still under development.